.github/workflows/tests_lkcpu_python.yml

```diff
@@ -180,6 +181,16 @@ jobs:
         run: |
           python -m pip install -U pennylane-lightning --no-deps
 
+      - name: Switch to stable tag of Lightning
+        if: inputs.lightning-version == 'stable'
+        run: |
+          cd main
+          git fetch --tags --force
+          git checkout $(git tag | sort -V | tail -1)
```

Comment:
Isn't this what `latest_release` was set to? The last released tag?

Reply:
The stable version is the one we push to pypi.
This version is based on the release branch.
The release branch has the tests, which should be for the stable version.

Reply:
I'm confused --- if we create a new tag, for any prupose, won't this influence this? I still don't see why `latest_release` cannot be used here. They give the same result, and the `latest_release` won't be moved if new tags are added:
```
git checkout $(git tag | sort -V | tail -1)
git checkout latest_release
```
Both give the same hash for me.

Reply:
Only tags that look like a release tag will be returned by this, and always the most recent.
But I agree that latest_release is equivalent.

Reply:
Just created a new tag starting with w though z and it superseded the expected order here.

Reply:
Yea, let's use `latest_release` instead, since that's depended upon for this purpose.

Reply:
Sure thing. I didn't give too much thought to that, because of time and the fact that this piece of code has been running around for some time. But you are right. I will also update it in all other workflows. Thanks!